pkg/ddc/jindofsx/transform.go

```diff
@@ -218,6 +222,11 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	// combine properties together
 	if len(runtime.Spec.Master.Properties) > 0 {
 		for k, v := range runtime.Spec.Master.Properties {
+			if k == "namespace.meta-dir" && runtime.Spec.Master.Replicas == 3 {
+				err = fmt.Errorf("Not support set namespace.meta-dir with %v with replicas = 3", v)
```

Comment:
error strings should not be capitalized (ST1005)

Reply:
ok